test/compilable/makedeps_lib.d

```diff
@@ -0,0 +1,27 @@
+/**
+DISABLED: win
+REQUIRED_ARGS: -makedeps -Jcompilable/extra-files -lib
+LINK:
+TRANSFORM_OUTPUT: remove_lines(druntime)
+TEST_OUTPUT:
+---
+$r:.*makedeps_lib_$0.$?:windows=lib|a$: \
+  $p:makedeps_lib.d$ \
+  $p:imports/makedeps_a.d$ \
+  $p:makedeps-import.txt$ \
+---
+**/
+// Disabling on windows because default naming of -lib seems broken (names to .exe)
```

Comment:
This should probably be looked into, even if probably all build systems specify `-of`.

Reply:
Do you suggest something specific for this test?
If I reactivate the test, it will fail on Windows, and fixing the default naming is out of scope of the PR.